src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,77 @@
+package duke.storage;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.util.Scanner;
+import java.util.List;
+import java.util.ArrayList;
+
+import duke.tasks.*;
```

Comment:
`import duke.tasks.*` should be replaced with the individual classes in the tasks package.